README.md

```diff
@@ -775,11 +778,127 @@ routes => [
 #####`type`
 *Optional* The type of VPN gateway. The only currently supported value --- and the default --- is 'ipsec.1'.
 
-#### Type: route53*
+#### Type: rds_db_parameter_group
+
+Note that this type is currently only available to list via puppet
+resource and cannot be created by Puppet.
+
+#####`name`
+*Required* The name of the parameter group.
+
+#####`region`
+*Required* The region in which to launch the parameter group. For valid values, see [AWS Regions](http://docs.aws.amazon.com/general/latest/gr/rande.html#ec2_region).
+
+#####`description`
+A description of the parameter group.
+
+#####`family`
+The name of the database family with which the parameter group is
+compatible. For instance mysql5.1.
+
+#### Type: rds_db_securitygroup
+
+#####`name`
+*Required* The name of the RDS DB security group.
+
+#####`description`
+A description of the RDS DB security group.
+
+#####`region`
+*Required* The region in which to launch the parameter group. For valid values, see [AWS Regions](http://docs.aws.amazon.com/general/latest/gr/rande.html#ec2_region).
+
+#####`owner_id`
+The internal AWS id of the owner of the security group. Read-only
```

Comment:
I take this to mean read-only ever, not read-only after creation? If so, looks fine, but needs a period after Read-only. Same for similar params below.